ingeniamotion/capture.py

```diff
@@ -107,12 +111,18 @@ def create_poller(
 
         Raises:
             IMRegisterNotExist: If register does not exist in dictionary.
+            TypeError: If some parameter has an error type.
```

Comment:
error type? I think it should be "wrong type". "If some parameter has a wrong type"

Reply:
That comment affects a lot of lines